components/authn-service/tokens/pg/sql/02_add__active__column_to_clients_table.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE chef_authn_clients ADD COLUMN active BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
🤔 why isn't github highlighting this like the others?

Reply:
I have no idea 😅